CHANGELOG.md

```diff
@@ -10,6 +10,12 @@
 
 - Add `time_nanos` to `BlockInfo` allowing access to high precision block times.
 - Change `FullDelegation::accumulated_rewards` from `Coin` to `Vec<Coin>`.
+- Rename `InitResponse::log`, `MigrateResponse::log` and `HandleResponse::log`
+  to `InitResponse::attributes`, `MigrateResponse::attributes` and
+  `HandleResponse::attributes`.
+- Rename `LogAttribute` to `Attribute`.
+- Remove `log` in favour of `Attribute::new`.
```

Comment:
The first names I agree with.
This seems a bit overly verbose. `attr()` instead of `log()` would be simpler. I know you like clarity, but lots of people have trouble with verbose APIs (and this is for contract devs, not vm developers who really appreciate clarity)

Reply:
Okay, will adapt